src/bluehawk/actions/snip.ts

```diff
@@ -11,7 +11,7 @@ export interface SnipArgs extends ActionArgs {
   state?: string;
   id?: string | string[];
   ignore?: string | string[];
-  format?: "rst";
+  format?: "rst" | "docusaurus";
```

Comment:
Hot take: 
```js
type Format = "rst" | "docusaurus";
type SnipArgs = {
   ...
  format?: Format | Format[]
}
...
const formats = args.format && !Array.isArray(args.format) ? [args.format] : args.format;
```
And then call createFormattedCodeBlock() for each format in the array (which could be none).

Reply:
Does this mean we would allow multiple formats? Unsure if that's something we should encourage/a valid use case.